crates/router/src/connector/applepay/transformers.rs

```diff
@@ -36,6 +37,19 @@ pub struct ErrorResponse {
     pub status_message: String,
 }
 
+#[derive(Debug, Default, Serialize, Deserialize)]
+pub struct ApplePayMetaData {
+    pub payment_object: PaymentObjectMetaData,
```

Comment:
Don't use object, give prefix or suffix with actual term which gives the meaning of what this struct do.